mls/nostr-mls/src/groups.rs

```diff
@@ -67,17 +87,75 @@ impl NostrGroupConfigData {
         image_url: Option<String>,
         image_key: Option<Vec<u8>>,
         relays: Vec<RelayUrl>,
+        admins: Vec<PublicKey>,
     ) -> Self {
         Self {
             name,
             description,
             image_url,
             image_key,
             relays,
+            admins,
         }
     }
 }
 
+impl NostrGroupDataUpdate {
+    /// Creates a new empty update configuration
+    pub fn new() -> Self {
+        Self {
+            name: None,
+            description: None,
+            image_url: None,
+            image_key: None,
+            relays: None,
+            admins: None,
+        }
+    }
+
+    /// Sets the name to be updated
+    pub fn name<T: Into<String>>(mut self, name: T) -> Self {
```

Comment:
Sorry for nitpicking, can you rewrite this?

```rust
pub fn name<T>(mut self, name: T) -> Self 
where
    T:  Into<String>,
{
    // ...
}
```

The same for the other methods below with generics